recipe/meta.yaml

```diff
@@ -25,6 +25,9 @@ build:
   - DPCPPROOT
   - DALROOT
   - NO_DIST=1  # [win]
+  ignore_run_exports:
+  - dpcpp_linux-64  # [linux64]
+  # - dpcpp_win-64  # [win]
 
 requirements:
```

Comment:
Is this exact same as scikit-learn-intelex meta.yaml? Is it possible to pull from there directly?